util/buildRelease/gen_release

```diff
@@ -146,6 +146,7 @@ if ($no_clone == 0) {
     "util/config",
     "util/quickstart",
     "util/test"
+    "tools/chplvis"
```

Comment:
Note that you could probably just say "tools" as well, which might be more future-proof when we have a second?